openedx/core/djangoapps/content_libraries/rest_api/serializers.py

```diff
@@ -172,6 +172,7 @@ class LibraryXBlockMetadataSerializer(PublishableItemSerializer):
     Serializer for LibraryXBlockMetadata
     """
     block_type = serializers.CharField(source="usage_key.block_type")
+    published_display_name = serializers.CharField()
```

Comment:
Nit: if you added this field to the `PublishableItem` dataclass, shouldn't it be on `PublishableItemSerializer` instead of `LibraryXBlockMetadataSerializer` ?

Fine to merge either way, just wondering.

Reply:
You're right. Updated: https://github.com/openedx/edx-platform/pull/36686/commits/0f781218c412f1ef6d16076bd5cb1ce2615fc64e